setuptools/build_meta.py

```diff
@@ -101,8 +101,12 @@ def _file_with_extension(directory, extension):
         f for f in os.listdir(directory)
         if f.endswith(extension)
     )
-    file, = matching
-    return file
+    try:
+        return next(matching)
```

Comment:
Previously, the code would check for exactly one match. Now it only checks for at least one match (and ignores all but the first). Could that silently swallow errors?

Reply:
I suppose, if multiple distributions would be built somehow or if the folder already had a distribution in it but then all bets are off anyway.